rootfs/etc/cont-init.d/14-config-rspamd.sh

```diff
@@ -12,10 +12,14 @@ if [ ! -f "$DKIM_PRIVATE_KEY" ]; then
   echo "WRN: $DKIM_PRIVATE_KEY not found. Rspamd service disabled."
   exit 0
 fi
+if [ -z "$ANONADDY_DKIM_SELECTOR" ]; then
+  export ANONADDY_DKIM_SELECTOR="default"
+  echo "INFO: DKIM selector not set, using default."
+fi
```

Comment:
I think we can just comment https://github.com/anonaddy/docker/blob/6cfc4152391439c458c9e0b1630f6806ca700762/rootfs/etc/cont-init.d/00-env#L77 and remove this change.